src/referencePackages/src/dir.props

```diff
@@ -26,5 +26,10 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <WarningsAsErrors></WarningsAsErrors>
+    <!--
+      NETStandard.Library is no longer built in 5.0, so the version does not exist in
+      PackageVersions.props.  Setting to latest version built in sbrp.
+    -->
+    <NETStandardLibraryPackageVersion>2.2.0-prerelease.19564.1</NETStandardLibraryPackageVersion>
```

Comment:
Why does this version make more sense than the latest that was actually shipped to the gallery, 2.0.3? (What is this used for?)

Reply:
I think 2.0.3 should work.  It is used in refeerence-package projects when including NETStandard.Library